@jules
```

Total in 415850 ms
Error: error building site: render: failed to render pages: open /home/runner/work/arran4.github.io/arran4.github.io/public/sitemap.xml: permission denied
Error: Process completed with exit code 1.
```

--- a/.github/workflows/gh-pages.yml
+++ b/.github/workflows/gh-pages.yml
@@ -84,6 +84,7 @@ jobs:
         run: |
           hugo mod npm pack
           npm install
+          rm -rf public
           hugo --minify --templateMetrics
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

--- a/.github/workflows/gh-pages.yml
+++ b/.github/workflows/gh-pages.yml
@@ -84,7 +84,7 @@ jobs:
         run: |
           hugo mod npm pack
           npm install
-          rm -rf public
+          sudo rm -rf public
           hugo --minify --templateMetrics
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

